moped-editor/src/views/projects/projectView/ProjectFunding/SubprojectFundingModal.js

```diff
@@ -118,29 +140,26 @@ const SubprojectFundingModal = ({
         </IconButton>
       </DialogTitle>
       <DialogContent>
-        <MaterialTable
-          columns={columns}
-          data={filteredData}
-          localization={{
-            body: {
-              emptyDataSourceMessage: (
-                <Typography>No FDUs available</Typography>
-              ),
+        <DataGridPro
+          sx={dataGridProStyleOverrides}
+          autoHeight
+          columns={dataGridColumns}
+          disableColumnMenu
+          rows={filteredData}
+          getRowId={(row) => row.fdu}
+          density="comfortable"
+          getRowHeight={() => "auto"}
+          initialState={{
+            pagination: {
+              paginationModel: { pageSize: PAGE_SIZE, page: 0 },
             },
           }}
-          options={{
-            ...(filteredData.length < 11 && {
-              paging: false,
-            }),
-            search: false,
-            toolbar: false,
-            tableLayout: "fixed",
-            selection: true,
-            rowStyle: typographyStyle,
-            pageSize: 10,
-            showSelectAllCheckbox: false,
-          }}
-          onSelectionChange={(rows) => setSelectedFdus(rows)}
+          pagination
+          pageSizeOptions={[PAGE_SIZE]}
+          hideFooter={filteredData.length < PAGE_SIZE}
```

Comment:
The previous table hid the pagination if there were 10 or less rows so I kept that around, and I'm on the fence. Here is what it looks like both ways.

**With pagination hidden when there are 10 or less results:**
<img width="1459" alt="Screenshot 2024-12-30 at 11 01 10 AM" src="https://github.com/user-attachments/assets/c0428b79-0f54-4220-b71f-d89ee8455591" />

**With pagination always showing:**
<img width="1459" alt="Screenshot 2024-12-30 at 11 01 25 AM" src="https://github.com/user-attachments/assets/929ded7d-48da-4a10-b1ec-a3c997d0f210" />

Reply:
yeah, I also think i prefer the pagination visible all the time. in addition to the record count i like when it's disabled on-screen because it's a little hint that sometimes there may be a lot of rows in the table.

lately i have been feeling down about hidden functionality in general—i feel like it leads to a more consistent UX and maybe fewer bugs.

Reply:
i agree with john and mateo!

Reply:
awesome y'all - thanks! I removed the conditional render. 🗑️ 👋